chapters/lit_review/angle_VR.tex

```diff
@@ -282,6 +282,59 @@ \subsubsection{Angular Biasing Using the Exponential Transform}
 evenly distributing the problems in problems with anisotropy.
 MCNP Weight Window Generator \textbf{cite MCNP manual here}
 
+
+The AVATAR method
+\cite{van_riper_generation_1995, van_riper_avatarautomatic_1997} used an
+approximation of the angular flux--without explicitly calculating it--to
+generate angle-dependent weight windows. It operated under the approximation
+that
+the angular flux was separable and symmetric about the average current vector.
+The angular flux was then calculated using
+a product of a deterministically-obtained
+scalar flux and an exponential function, derived from the
+maximum entropy distribution, that was a function of the scalar flux and the
+current. Space-, energy-, and angle-dependent weight windows for
+the Monte Carlo problem were then generated from the inverse of the angular
+flux. AVATAR improved the FOM for sample problems from 2 to 5 times, but did not
```

Comment:
Yep! I'm pretty sure that's right (re: hard to use). This section was moved from Methodology to Lit Review so it's a bit light on details. When I go back through the actual lit review all three of these methods will be described w/ more detail.